app/controllers/movies_controller.rb

```diff
@@ -1,4 +1,6 @@
 class MoviesController < ApplicationController
   def index
+    #@movies = Movie.all
+    @movies = Movie.where(genre: ["basic", "git", "ruby", "rails"])
```

Comment:
`RAILS_GENRE_LIST` を利用しましょう。

```suggestion
    @movies = Movie.where(genre: Movie::RAILS_GENRE_LIST)
```